lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+require_relative "order"
+
```

Comment:
Good idea to have menu as a separate class. Maybe it could have had an option to inititalize with different menus.